pkg/cloudprovider/providers/azure/blobDiskController.go

```diff
@@ -0,0 +1,1036 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
actually it is 2016 in all the files i have checked. Wouldn't that break Bazel if changed?